lib/gringotts/gateways/trexle.ex

```diff
@@ -165,28 +179,39 @@ defmodule Gringotts.Gateways.Trexle do
   ## Example
   The following session shows how one would store a card (a payment-source) for future use.
   ```
-  iex> card = %{
-    name: "John Doe",
+  iex> card = %CreditCard{
     number: "5200828282828210",
-    expiry_month: 1,
-    expiry_year: 2018,
-    cvc: "123",
-    address_line1: "456 My Street",
-    address_city: "Ottawa",
-    address_postcode: "K1C2N6",
-    address_state: "ON",
-    address_country: "CA"
+    month: 12,
+    year: 2018,
+    first_name: "John",
+    last_name: "Doe",
+    verification_code: "123",
+    brand: "visa"
   }
 
-  iex> options = [email: "john@trexle.com", ip_address: "66.249.79.118", description: "Store Purchase 1437598192"]
+  iex> @address %Address{
+    street1: "123 Main",
+    street2: "Suite 100",
+    city: "New York",
+    region: "NY",
+    country: "US",
+    postal_code: "11111",
+    phone: "(555)555-5555"
+  }
+
+  iex> options = [email: "john@trexle.com", ip_address: "66.249.79.118", billing_address: @address, description: "Store Purchase 1437598192"]
 
   iex> Gringotts.store(:payment_worker, Gringotts.Gateways.Trexle, card, options)
   ```
   """
 
-  @spec store(map, list) :: map
+  @spec store(CreditCard.t, list) :: map
   def store(payment, opts \\ []) do
-    params = [email: opts[:email]]++card_params(payment)
+    params = [
+              email: opts[:email]
```

Comment:
no need to separate email in three different lines, you can do something like this

```
  [email: opts[:email]]
 ++ card_params(payment)
 ++ address_params(opts[:billing_address])
```